contracts/AuthenticatedTracker.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity 0.5.4;
```

Comment:
Tronbox only has support for certain solidity versions. I can bump this to 0.5.8 now and higher in the future.

Reply:
I think it's fine as is 🤝